maven-api-impl/src/main/java/org/apache/maven/internal/impl/model/MavenBuildTimestamp.java

```diff
@@ -70,7 +67,6 @@ public MavenBuildTimestamp(Instant time, String timestampFormat) {
         }
         SimpleDateFormat dateFormat = new SimpleDateFormat(timestampFormat);
         dateFormat.setCalendar(new GregorianCalendar());
-        dateFormat.setTimeZone(DEFAULT_BUILD_TIME_ZONE);
```

Comment:
Attention, this is a semantic change. We should retain `Etc/UTC` and maybe make it configurable in the future.

Reply:
Why is that a change ? We added the timezone offset in the description instead of using `Z`, so we need to use the current time zone, else, the formatted time will be wrong I think.

Reply:
Without setting the timezone it will use the local timezone. It won't be reproducible for those crossing time zones. If you set the timezone `XXX` will produce stable output, with `Etc/UTC` always `Z`. Then if we say that the timezone can be configured `XXX` will produce proper output, `'Z'` is just a static literal.

Reply:
https://reproducible-builds.org/docs/timezones/

Try it out in a `main()` and you will see the difference.